gradle.properties

```diff
@@ -0,0 +1,2 @@
+generator.version = 1
```

Comment:
I know it's just a detail but, is it `1` or `v1`? in the ticket it is 1 but in the kickoff document it is v1

Reply:
@ouvreboite what do you think?